_3_design_patterns/src/main/java/code/_3_in_class/Main.java

```diff
@@ -1,8 +1,9 @@
 package code._3_in_class;
 
-public class Main {
+public class HelloWorld {
```

Comment:
Observ ca ai redenumit clasa existenta "Main" in numele "HelloWorld".
Rugamintea este sa creezi o clasa noua si nu sa schimbi numele unei clase existente.